pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -236,3 +253,9 @@ type bytesStringer []byte
 func (b bytesStringer) String() string {
 	return string(b)
 }
+
+func isStandalone() bool {
+	value, set := os.LookupEnv("DD_APM_TRACING_ENABLED")
+	enabled, _ := strconv.ParseBool(value)
+	return set && !enabled
+}
```

Comment:
There is an identical exported function in `serverless/appsec`, it should be used instead of duplicated.

Reply:
Refactored it in the new `config` pkg, now used in `cmd/serverless` and internally. This is to avoid circular dependencies.
See 25c20caf62ffdde for the new test.